src/api/app/controllers/webui/project_controller.rb

```diff
@@ -24,6 +24,8 @@ class Webui::ProjectController < Webui::WebuiController
   before_filter :load_releasetargets, :only => [ :show, :incident_request_dialog, :release_repository_dialog ]
   prepend_before_filter :lockout_spiders, :only => [:requests, :rebuild_time, :buildresults, :maintenance_incidents]
 
+  after_action :verify_authorized, :only => :save_new
```

Comment:
I see. That's not the authorization call but a check to ensure authorize got called by the controller.